pkg/kube/eve-bridge/eve-bridge.go

```diff
@@ -0,0 +1,501 @@
+// Copyright (c) 2023 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"net"
+	"net/rpc/jsonrpc"
+	"os"
+	"strings"
+
+	"gopkg.in/natefinch/lumberjack.v2"
+
+	"github.com/containernetworking/cni/pkg/invoke"
+	"github.com/containernetworking/cni/pkg/skel"
+	"github.com/containernetworking/cni/pkg/types"
+	v1 "github.com/containernetworking/cni/pkg/types/100"
+	"github.com/containernetworking/cni/pkg/version"
+	"github.com/containernetworking/plugins/pkg/ipam"
+	bv "github.com/containernetworking/plugins/pkg/utils/buildversion"
+	"github.com/lf-edge/eve/pkg/kube/cnirpc"
+)
+
+const (
+	eveKubeNamespace       = "eve-kube-app"
+	zedrouterRPCSocketPath = "/run/zedrouter/rpc.sock"
+	primaryIfName          = "eth0"
+	clusterSvcIPRange      = "10.43.0.0/16"
+	vmiPodNamePrefix       = "virt-launcher-"
+)
+
+const (
+	logfileDir    = "/persist/newlog/kube/"
```

Comment:
Does newlogd know about this directory? How do we manage space there?

Reply:
Naiming moved all Kubernetes related logs to this directory: https://github.com/naiming-zededa/eve/pull/238
@naiming-zededa Could you please answer the question from Erik.
But as I understand it, we do not publish these logs (yet). For now all these logs are just saved in this dir and rotated to limit the size: https://github.com/naiming-zededa/eve/blob/307a8d72f85c9c1229bf4ba36da34a5fbf19169e/pkg/kube/cluster-init.sh#L32-L45

Reply:
@eriknordmark we have basically two types of logging from the kube containers. One is container level to the normal output, which comes to newlog, and it's too much logging, so currently in the newlogd in poc branch, we filter them out except the severity is error or warning.
The second type is the apps, generate log files, currently we have k3s.log, multus.log, k3s-install.log and eve-bridge.log. We keep all those logs, up to 5M, and rotate them and keep max of 3 copies of those logs for each type.

Reply:
Thanks @naiming-zededa